02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,95 @@
+						<ul>
+							<li>Responsible for pre-sales and post-sales support for LTE and 3G cellular test solutions at AT&T and Verizon.</li>
+							<li>Installing and maintaining complex test equipment used for device acceptance testing at telecom operators.</li>
+							<li>Develop Perl test scripts, and automated reports for detailed analysis of various handset KPIs.</li>
+							<li>Analyzing device performance issues with diagnostic monitoring tools to identify their root causes.</li>
+							<li>Resolving LTE and 3G network, handset, Android, and iOS handsets issues.</li>
+							<li>Automating handset operation, such as FTP, phone calls, SMS, web browsing, and Youtube.</li>
+							<li>Familiarity with debugging network related issues, both wired and wireless.</li>
+							<li>Compiling data and generating reports at customer labs to update engineering team of status of projects and issues.</li>
+						</ul>
+						<u>APAC Business Development Manager, Shanghai, China</u>
+						<ul>
+							<li>Responsible for the commercial development of LTE and 3G test solutions for handset and network testing with telecom operators in Asia.
+							<li>Led engineering team that developed handset testing solutions for 2 major telecom operators in Asia, docomo and China Mobile.
+							<li>Worked closely with telecom operators to identify ecosystem pull-through opportunities with telecom operator suppliers and test labs.
+							<li>Managed projects in both the development and on-site verification phases.
+							<li>Senior Field Application Engineer, Asia Pacific
+							<li>Responsible for pre-sales and post-sales support of cellular and Wi-Fi test solutions in APAC region.
+							<li>Opened first field office APAC region and trained and managed a team of distributors across APAC to be able to provide first line technical support and demonstrations.
+							<li>Demonstrated HW and SW solutions to major players in the wireless field customers with their own with unique testing requirements.
+							<li>Compiled data about competing products in order for the sales team to better position the company and products better against the competition.
+							<li>Developed automated solutions to expedite customer testing and expand product value proposition..
+							<li>Debugging technical issues related to TCP/IP, WAN, LAN, 802.11a/b/g/n, VoIP, WiMAX, and LTE.
+							<li>Familiarity with operating 3rd party test equipment such as network analyzers, spectrum analyzers, and base station emulators.
+						</ul>
+					<h4>LANGUAGES</h4>
+          <ul>
+	          <li>English - Native</li>
+	          <li>Chinese, Mandarin - Conversational</li>
+	          <li>Japanese - Basic (Currently studying)</li>
+          </ul>
+	</div>
+
+	<div align = "center"><a href="https://www.austincodingacademy.com"><img src="https://www.austincodingacademy.com/wp-content/uploads/2016/05/Banner-Logo-2-2.png"></a></div>
+	</body>
 
-</body>
 </html>
```

Comment:
Outstanding work. While this doesn't actually follow the specific guidelines for the homework, you have proven a level of mastery to the essence of the assignment. Well done.